src/Cli/dotnet/commands/dotnet-test/VSTestFeatureFlag.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
```

Comment:
Nit: 

```suggestion
```

Reply:
It's weird...I've used auto format inside VS and it added this.